src/main/java/org/retest/rebazer/connector/BitbucketConnector.java

```diff
@@ -57,15 +71,7 @@ String getHeadOfBranch( final PullRequest pullRequest ) {
 	}
 
 	String getLastCommonCommitId( final PullRequest pullRequest ) {
```

Comment:
Does it make sense to write tests for PRs with < 10 and >= 10 commits to verify the new behavior, or is it too much effort?

Reply:
We should definitely do this - in the future.